api-tests/features/p1-journeys.feature

```diff
@@ -0,0 +1,248 @@
+@Build
+Feature: P2 journey
+
+  Scenario: P1 App Journey
+    Given I start a new 'low-confidence' journey with feature set 'p1Journeys'
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'appTriage' event
+    Then I get a 'dcmaw' CRI response
+    When I submit 'kenneth-driving-permit-valid' details to the CRI stub
```

Comment:
There are a couple of places like this where the e2e test used unrealistic score values. I think using different realistic values here is fine, but it's not exactly the same as the test it's replacing.